### PR TITLE
En 6082 process pools cleaner unit tests

### PR DESCRIPTION
**Pull request by miiu96**

### CODE REVIEW

File: process/mock/addressConverterStub.go [hunk 1]
```diff
@@ -35,7 +35,10 @@ func (acs *AddressConverterStub) PrepareAddressBytes(addressBytes []byte) ([]byt
 
 // AddressLen -
 func (acs AddressConverterStub) AddressLen() int {
```
Comment (SebastianMarian) [inline process/mock/addressConverterStub.go:37]: add pointer receiver *AddressConverterStub, and maybe rename AddressLenHandler to AddressLenCalled to be in the same format with the other methods in this file

File: process/block/poolsCleaner/crossTxsPoolsCleaner_test.go [hunk 2]
```diff
@@ -1,3 +1,391 @@
...
+	tx := &transaction.Transaction{
+		SndAddr: sndAddr,
+	}
+	crossTxCleaner.receivedUnsignedTx(txKey, tx)
+
+	numTxsInMap := crossTxCleaner.cleanCrossTxsPoolsIfNeeded()
+	assert.Equal(t, 0, numTxsInMap)
+}
+
+func TestCleanCrossTxsPoolsIfNeeded_RoundDiffToSmallShouldNotBeRemoved(t *testing.T) {
```
Comment (SebastianMarian) [inline process/block/poolsCleaner/crossTxsPoolsCleaner_test.go:305]: TooSmall

File: process/block/poolsCleaner/crossTxsPoolsCleaner_test.go [hunk 3]
```diff
@@ -1,3 +1,391 @@
...
+	tx := &transaction.Transaction{
+		SndAddr: sndAddr,
+	}
+	crossTxCleaner.receivedUnsignedTx(txKey, tx)
+
+	numTxsInMap := crossTxCleaner.cleanCrossTxsPoolsIfNeeded()
+	assert.Equal(t, 1, numTxsInMap)
+}
+
+func TestCleanCrossTxsPoolsIfNeeded_RoundDiffToBigShouldBeRemoved(t *testing.T) {
```
Comment (SebastianMarian) [inline process/block/poolsCleaner/crossTxsPoolsCleaner_test.go:343]: TooBig

File: process/block/poolsCleaner/miniBlocksPoolsCleaner_test.go [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+	key := []byte("mbKey")
+	miniblock := &block.MiniBlock{}
+	miniblockCleaner.receivedMiniBlock(key, miniblock)
+
+	result := miniblockCleaner.cleanMiniblocksPoolsIfNeeded()
+	assert.Equal(t, 0, result)
+}
+
+func TestCleanMiniblocksPoolsIfNeeded_RoundDiffToShouldMiniblockShouldRemainInMap(t *testing.T) {
```
Comment (SebastianMarian) [inline process/block/poolsCleaner/miniBlocksPoolsCleaner_test.go:87]: Refactor the name of this method

File: process/mock/addressConverterStub.go (hunk 1 above)
Comment (miiu96) [reply process/mock/addressConverterStub.go:37]: done.

File: process/block/poolsCleaner/crossTxsPoolsCleaner_test.go (hunk 2 above)
Comment (miiu96) [reply process/block/poolsCleaner/crossTxsPoolsCleaner_test.go:305]: done.

File: process/block/poolsCleaner/crossTxsPoolsCleaner_test.go (hunk 3 above)
Comment (miiu96) [reply process/block/poolsCleaner/crossTxsPoolsCleaner_test.go:343]: done.

File: process/block/poolsCleaner/miniBlocksPoolsCleaner_test.go (hunk 4 above)
Comment (miiu96) [reply process/block/poolsCleaner/miniBlocksPoolsCleaner_test.go:87]: done.

File: process/block/poolsCleaner/miniBlocksPoolsCleaner_test.go [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+func TestCleanMiniblocksPoolsIfNeeded_MiniblockNotInPoolShouldBeRemovedFromMap(t *testing.T) {
+	t.Parallel()
+
+	miniblockCleaner, _ := NewMiniBlocksPoolsCleaner(&mock.CacherStub{
+		GetCalled: func(key []byte) (value interface{}, ok bool) {
+			return nil, false
+		},
+	}, &mock.RounderMock{}, &mock.CoordinatorStub{})
```
Comment (bogdan-rosianu) [inline process/block/poolsCleaner/miniBlocksPoolsCleaner_test.go:77]: put each mock on a new line. Apply this in the entire PR, please

Comment (miiu96) [reply process/block/poolsCleaner/miniBlocksPoolsCleaner_test.go]: done

**Pull request merged**